app/controllers/support_interface/new_provider_users_controller.rb

```diff
@@ -0,0 +1,86 @@
+module SupportInterface
+  class NewProviderUsersController < SupportInterfaceController
+    def new
+      @provider = Provider.find(provider_id_param)
+      @form = NewProviderUserForm.new(provider_id: provider_id_param)
```

Comment:
Same here re: form name
https://github.com/DFE-Digital/apply-for-teacher-training/pull/4828/files#r639814295

Reply:
There are two forms for a single provider user - one for creating and another for editing (the latter accommodates all of a provider user's permissions). I've renamed the forms CreateSingleProviderUserForm  and EditSingleProviderUserForm for now. We can revisit once we start the bulk upload